Automattic/wp-calypso
# Add url locale on stepper

## Proposed Changes

**For not logged in users**, if there is a `&locale=XX` param in the url, the flow should be loaded in that language.
**For logged in users**, we their own locale.

## Testing Instructions

 - Pull this branch and run yarn start from the root folder
 - Navigate to `/setup/intro?flow=newsletter` or `/setup/intro?flow=link-in-bio`
 - You should see the app in your [default language ](https://wordpress.com/me/account) "Interface Language"
 - Add `&locale=XX` you should see on the specific language if supported  


<img width="1385" alt="image" src="https://user-images.githubusercontent.com/2653810/188642075-80634640-67d9-486c-850b-568c65e4c3f1.png">

<img width="1388" alt="image" src="https://user-images.githubusercontent.com/2653810/188642169-3576e9c6-ca86-4a4c-8e5f-0efade895d71.png">

<img width="1387" alt="image" src="https://user-images.githubusercontent.com/2653810/188642286-f786530f-706f-420c-84fc-24e2f5055935.png">



Related to #67366
Fixes #67366
